modules/op-sample-example-pipeline-tutorial.adoc

```diff
@@ -0,0 +1,49 @@
+[id='op-sample-example-pipeline-tutorial_{context}']
+= Sample example from pipeline tutorial
+
+We are using https://github.com/openshift/pipelines-tutorial[pipelines tutorial] as a sample example.
+
+.Procedure
+
+. Create the `TriggerBinding` resource directly from the pipelines-tutorial repository:
++
+[source,terminal]
+----
+oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/01_binding.yaml
+----
+
+. Create the `TriggerTemplate` resource directly from the pipelines-tutorial repository:
++
+[source,terminal]
+----
+oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/02_template.yaml
+----
+
+. Create the Trigger resource directly from the pipelines-tutorial repository:
++
+[source,terminal]
+----
+oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/03_trigger.yaml
+----
+
+. To create an EventListener resource using a secure HTTPS connection:
```

Comment:
Don't start with "To". Maintain similarity with the previous steps, even if this one contains sub-steps.

Create an `EventListener` resource using a secure HTTPS connection: